lib/dataset-tools/augmentation/replace_parameters.js

```diff
@@ -260,7 +260,7 @@ module.exports = class ParameterReplacer {
             slot.tag === 'program.principal'))
             return ['string', 'tt:person_first_name'];
 
-        if (!slot.type.isEntity && !slot.type.isString && !slot.type.isLocation && !slot.type.isNumber)
+        if (!slot.type.isEntity && !slot.type.isString && !slot.type.isLocation && !slot.type.isNumber && !slot.type.isMeasure)
```

Comment:
Do we handle measurements correctly though?

Reply:
They are currently treated the way numbers are treated (assuming measurement units are included in the original sentence)

Reply:
Ok